spec/queryrenderer/AggregateFunctionAppender_spec.lua

```diff
@@ -36,11 +36,56 @@ describe("AggregateFunctionRenderer", function()
                     {type = "column", name =  "customer_id", tableName = "customers"}),
             "APPROXIMATE_COUNT_DISTINCT")
 
+    it_asserts("COUNT(*)", run_function("COUNT"), "COUNT(*)")
+
     for _, grouping_alias in ipairs({"GROUPING", "GROUPING_ID"}) do
         it_asserts(grouping_alias .. '("sales"."yr", "sales"."mon")',
                 run_function(grouping_alias,
                         {type = "column", name =  "yr", tableName = "sales"},
                         {type = "column", name =  "mon", tableName = "sales"}),
                 grouping_alias)
     end
+
+    for _, single_parameter_functions_without_distinct in ipairs({
+        "FIRST_VALUE", "LAST_VALUE", "MEDIAN"
+    }) do
+        local expression = {
+            type = "predicate_less",
+            left = {type = "column", name = "age", tableName = "visitors"},
+            right = literal.exactnumeric(30)
+        }
+        it_asserts(single_parameter_functions_without_distinct .. '(("visitors"."age" < 30))',
```

Comment:
Using double parenthesis in `'(("visitors"."age" < 30))'` by intention?

Reply:
As discussed we keep it like this, since always putting parenthesis around binary predicates makes the code simpler.